Lib/pathlib.py

```diff
@@ -390,7 +390,7 @@ class PurePath(object):
         '_str', '_hash', '_pparts', '_cached_cparts',
     )
 
-    def __new__(cls, *args):
+    def __new__(cls, *args, **kwargs):
```

Comment:
We don't want to allow keyword arguments, we want them to be an error. Doing this will make this no longer be the case. The code as in Python 3.11 already raises an exception:

```python
>>> pathlib.PurePath(a='a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: PurePath.__new__() got an unexpected keyword argument 'a'
```

Reply:
Fixed, now `pathlib.Path` and `pathlib.PurePath` doesn't have `**kwargs` in their signatures.